packages/plugin-js-packages/src/lib/constants.ts

```diff
@@ -1,7 +1,13 @@
 import { IssueSeverity } from '@code-pushup/models';
-import type { DependencyGroup, PackageAuditLevel } from './config';
+import type {
+  DependencyGroup,
+  PackageAuditLevel,
+  PackageManagerId,
+} from './config';
 import { DependencyGroupLong } from './runner/outdated/types';
 
+export const DEFAULT_PACKAGE_MANAGER: PackageManagerId = 'npm';
```

Comment:
👨‍💻 Naming: Not sure default is a good name - it is only used if attempts to find out the package manager fail. `FALLBACK_PACKAGE_MANAGER` might be better.

Reply:
done!